server/ai_http.go

```diff
@@ -96,26 +97,95 @@ func (h *lphttp) StartLiveVideoToVideo() http.Handler {
 
 		remoteAddr := getRemoteAddr(r)
 		ctx := clog.AddVal(r.Context(), clog.ClientIP, remoteAddr)
+		requestID := string(core.RandomManifestID())
 
-		// skipping handleAIRequest for now until we have payments
-
-		var (
-			mid        = string(core.RandomManifestID())
-			pubUrl     = TrickleHTTPPath + mid
-			subUrl     = pubUrl + "-out"
-			controlUrl = pubUrl + "-control"
-		)
-		jsonData, err := json.Marshal(
-			&worker.LiveVideoToVideoResponse{
-				PublishUrl:   pubUrl,
-				SubscribeUrl: subUrl,
-				ControlUrl:   controlUrl,
-			})
+		payment, err := getPayment(r.Header.Get(paymentHeader))
 		if err != nil {
+			respondWithError(w, err.Error(), http.StatusPaymentRequired)
+			return
+		}
+		sender := getPaymentSender(payment)
+
+		var req worker.GenLiveVideoToVideoJSONRequestBody
+		if err := jsonDecoder[worker.GenLiveVideoToVideoJSONRequestBody](&req, r); err != nil {
+			respondWithError(w, err.Error(), http.StatusBadRequest)
+			return
+		}
+
+		orch := h.orchestrator
+		pipeline := "live-video-to-video"
+		cap := core.Capability_LiveVideoToVideo
+		modelID := *req.ModelId
+		clog.V(common.VERBOSE).Infof(ctx, "Received request id=%v cap=%v modelID=%v", requestID, cap, modelID)
+
+		// Create storage for the request (for AI Workers, must run before CheckAICapacity)
+		err = orch.CreateStorageForRequest(requestID)
```

Comment:
This is required for the AI Worker downstream to process the request (I tested without it), It is boilerplate on all the pipelines and might need some refactoring later to make it optional since we are not expecting a response back necessarily.